include/RAJA/policy/cuda/policy.hpp

```diff
@@ -179,14 +180,10 @@ using cuda_reduce_atomic_async = cuda_reduce_atomic<BLOCK_SIZE, true>;
 // values for CUDA warp size and max block size.
 //
 const int WARP_SIZE = 32;
-const int RAJA_CUDA_MAX_BLOCK_SIZE = 2048;
-
-/*!
- * \def RAJA_CUDA_LAUNCH_PARAMS(gridSize, blockSize)
- * Macro that generates kernel launch parameters.
- */
-#define RAJA_CUDA_LAUNCH_PARAMS(gridSize, blockSize) \
-  gridSize, blockSize, getCudaSharedmemAmount(gridSize, blockSize)
+const int RAJA_CUDA_MAX_BLOCK_SIZE = 1024;
```

Comment:
2) Maybe, this might be something that could be configured by looking at the sm version provided.